app/src/components/CopiableCode.tsx

```diff
@@ -0,0 +1,43 @@
+import { HStack, Icon, IconButton, Tooltip, Text } from "@chakra-ui/react";
+import { useCallback, useState } from "react";
+import { MdContentCopy } from "react-icons/md";
+
+const CopiableCode = ({ code }: { code: string }) => {
+  const [copied, setCopied] = useState(false);
+
+  const copyToClipboard = useCallback(() => {
+    const onCopy = async () => {
```

Comment:
[Is](url) this inner function just to avoid the lint errors if you have an async `useCallback`? If so why not use `useHandledAsyncCallback`?

Reply:
Good idea! That's much cleaner.